mk/config.mk

```diff
@@ -222,6 +222,10 @@ CFG_WITH_PAGER ?= n
 # Use the pager for user TAs
 CFG_PAGED_USER_TA ?= $(CFG_WITH_PAGER)
 
+ifeq ($(CFG_WITH_PAGER),y)
```

Comment:
Let's do it unconditionally and add a comment (say that it reduces the fragmentation of the TEE core heap)

Reply:
This is a bit more flexible, +1 to a comment though.
A more compact way of writing this is:
`CFG_BGET_BESTFIT ?= $(CFG_WITH_PAGER)`